File: Java/ArraysBidimensionals/src/ex02.java
public class ex02 {

	public static void main (String[] args){

		String[][] matrix = new String[5][5];

		for (int i = 0; i < matrix.length; i++) {

			for (int j = 0; j < matrix[i].length; j++) {

				matrix[i][j]="(" + i + "," + j + ")";
			} 
		} 

		int i = 0;
		int j = 0;

		while (i < matrix.length) {
			j = 0;

			while(j < matrix[i].length){

				System.out.print (matrix[i][j]);
				j++;
			}
			System.out.println ();
			i++;
		}

	}

}

File: Base de dades/Php/PhpProject1/PhpProject1/ex02.php
<html>
    <head>
        <meta charset="UTF-8">
        <title>ex02</title>
    </head>
    <body>

        <?php
        // Escribe el código que a partir de dos variables $numero1 y $numero2 indique cuál es mayor y cual es menor. En caso de empate debe indicar que son iguales. También debe mostrar el resultado de sumarlos y restarlos.

        $num = 5;
        $num2 = 6;

        if ($num > $num2) {
            echo "$num Es mayor que $num2 <br>";
        } else if ($num == $num2) {
            echo "$num Es igual que $num2 <br>";
        } else if ($num < $num2) {
            echo "$num Es menor que $num2 <br>";
        } else {
            echo"numero erroneo<br>";
        }
        $suma = $num + $num2;
        $resta = $num - $num2;
        echo "La suma de $num+$num2 es = $suma <br>";
        echo "La resta de $num-$num2 es = $resta <br>";
        ?>
    </body>
</html>